dist/index.html

```diff
@@ -0,0 +1,39 @@
+<html>
```

Comment:
Can you move this to /demo?

Reply:
Do you want this in a /demo folder at root or in a folder at /dist/demo?

If `npm run demo` command is reading from a `/demo` folder it won't have access to the js files in dist. Maybe I could do a symlink?